### DISCUSSION TITLE
Does it support Bootstrap 5 ?

### DISCUSSION THREAD

**Discussion by bacloud22** [julien-deramond/bootstrap-deprecated-classes-extension#5] [Q&A] [answered]
Hi, 
I'm using latest version of Bootstrap (5). I've download data and built after for *Chrome*
I've run it and it shows in the contextual menu only bootstra 4 / 3 only and no deprecated classes are highlighted.

**Comment by julien-deramond** [comment, marked as answer]
Hi there,

The extension basically currently compares Bootstrap v5 classes to the latest v4 and latest v3 classes. Said differently, if you use the extension against a website developed with some deprecated classes from Bootstrap v4 or v3, they will be highlighted.
Otherwise, if the website is developed in Bootstrap v5 perfectly, nothing will be highlighted.

On Chrome, you can install it [via this link](https://chromewebstore.google.com/detail/bootstrap-deprecated-clas/cfkfbhnancjclhapldjkpllagfjpdihc).
Then, if you go to https://getbootstrap.com/docs/4.6/utilities/colors/#color, and click on "Bootstrap v4" in the contextual menu, you'll have some highlighted elements in the browser:

<img width="1464" alt="Screenshot 2024-06-06 at 17 50 45" src="https://github.com/julien-deramond/bootstrap-deprecated-classes-extension/assets/17381666/3c68b579-d89c-4c29-acc3-7384af1c756a">

**Reply by bacloud22** [reply]
Thank you so much for your response. It does work, I was simply missing those deprecated in some pages and present in others so I didn't pay attention.
